httpd/handler_test.go

```diff
@@ -87,13 +87,13 @@ func TestBatchWrite_UnmarshalEpoch(t *testing.T) {
 		t.Logf("testing %q\n", test.name)
```

Comment:
This would be a good spot to add a quick test for sending in no timestamp and making sure it comes back as a zero time.